module5/assignment5.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="module4/_css/kotz_a5.css"> 
+		<link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+		<script src="http://code.jquery.com/jquery-2.1.3.min.js"></script>
+		<script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+	</head>
+	<body>
+		<!-- Collin Kotz
+			.html
+			INFO2439
+			Thoendel
+			10/13/19
+		-->
+		
+		<div data-role="page" id="map-page" data-title="The MCC Sage Bistro">
+			<div data-role="header" id="ui-content">
+				<h1>Where To Find Us</h1>
+			</div>
+
+			<img src="https://maps.googleapis.com/maps/api/staticmap?center=41.304729,-95.959506&zoom=17&size=150x150&maptype=roadmap
+			&markers=color:blue%7Clabel:S%7C41.304729,-95.959506&key=AIzaSyAFQid4GkVK5cIqrk3n8kuRkAo7YcNeTzo">
```

Comment:
This looks like you used my API key. I would then expect some screenshots either here in your repo or in your BB submission showing that you created your own credentials without billing attached.